compoments/radioFieldSet.js

```diff
@@ -73,6 +74,7 @@ class RadioFieldSet extends Component {
                 {this.title}
               </h1>
             </legend>
+            { this.hintText && <div id={`hint-text-${this.name}`} className='govuk-hint'>{this.hintText}</div>}
```

Comment:
is this component used elsewhere, if so do we want this change to be reflected in all instances where this component is used?

Reply:
It is used elsewhere, which is why I have made it optional i.e. it won't render if a value isn't passed to it.